src/processes.c

```diff
@@ -2176,8 +2175,9 @@ static int ps_read(void) {
     ps_submit_proc_list(ps_ptr);
 
   read_fork_rate(buffer);
-  read_sys_ctxt_switch(buffer);
-  /* #endif KERNEL_LINUX */
+  if (report_sys_ctxt_switch)
+    read_sys_ctxt_switch(buffer);
+    /* #endif KERNEL_LINUX */
```

Comment:
Is this (commented `#endif`) useful ?

Reply:
I guess this makes it clearer where the end of the "KERNEL_LINUX" part is. Since this kind of comment exists in other places, e.g., #319, #786, maybe we should keep it for this PR?

BTW, the starting spaces are caused by format.sh